ibis_substrait/compiler/translate.py

```diff
@@ -474,12 +476,25 @@ def value_op(
     compiler: SubstraitCompiler,
     **kwargs: Any,
 ) -> stalg.Expression:
+    error_args = []
+    # TODO(gforsyth): remove this brittle workaround after extension parsing is ready
+    if (
+        isinstance(op, ops.Binary)
+        and not isinstance(op, ops.Comparison)
+        and isinstance(op.left.type(), (dt.Integer, dt.Floating, dt.Decimal))
+        and isinstance(op.right.type(), (dt.Integer, dt.Floating, dt.Decimal))
```

Comment:
```suggestion
        isinstance(op, ops.NimericBinary)
```
or [maybe](https://github.com/ibis-project/ibis/blob/4df30e61862d436f5966ca3eb2381e6941d485bc/ibis/expr/operations/numeric.py#L19)
```suggestion
        isinstance(op, (ops.Add, ops.Multiply, *other_error_needing_ops))
```

Reply:
I went with `NumericBinary` and also added in a check to make sure we add in the required `rounding` argument for non-integers

Reply:
Scratch that -- `NumericBinary` was added after Ibis 3.0.2